File: Sve.WebHost/ClientApp/src/app/admin/accounts/_models/transactions.ts
export class TransactionsModel {
	VoucherTypeId: number;
		AccountTypeId: number;
		CustomerId: number;
		PayModeId: number;
		TransactionId: number;
		PaidAmount: number;
		PaidDate: any;
		Remarks: string;
		CreatedOn: any;
		CreatedBy: string;
		ModifiedOn: any;
		ModifiedBy: string;
		Status: number;
	
	clear(): void {
			this.VoucherTypeId = null;
			this.AccountTypeId = null;
			this.CustomerId = null;
			this.PayModeId = null;
			this.TransactionId = null;
			this.PaidAmount = null;
			this.PaidDate = new Date();
			this.Remarks = '';
			this.CreatedOn = null;
			this.CreatedBy = '';
			this.ModifiedOn = null;
			this.ModifiedBy = '';
			this.Status = null;
		}
}

File: Sve.Contract/Models/Accounts/Transactions.cs

namespace Sve.Contract.Models.Accounts
{
	using System;
	using System.Collections.Generic;
	using System.ComponentModel.DataAnnotations;

    public class Transactions
    {
		public int? VoucherTypeId { get; set; }
		public int? AccountTypeId { get; set; }
		public int? CustomerId { get; set; }
		public int? PayModeId { get; set; }
        public long TransactionId { get; set; }
		public decimal? PaidAmount { get; set; }
		public DateTime? PaidDate { get; set; }
		public string Remarks { get; set; }
		public short? Status { get; set; }

		public virtual AccountType AccountType { get; set; }
		public virtual Customer Customer { get; set; }
		public virtual PayMode PayMode { get; set; }
		public virtual VoucherType VoucherType { get; set; }
		public virtual ICollection<TransactionDetail> TransactionDetails { get; set; }
        public DateTime? CreatedOn { get; set; }
    }
}